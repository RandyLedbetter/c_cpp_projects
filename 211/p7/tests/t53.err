Error: could not insert Sacramento, Redding
