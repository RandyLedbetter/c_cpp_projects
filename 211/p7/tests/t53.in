insert Chico Sacramento 90
insert Chico Redding 73
insert Sacramento San_Francisco 87
insert Sacramento Redding 163
lookup Chico
lookup Sacramento
