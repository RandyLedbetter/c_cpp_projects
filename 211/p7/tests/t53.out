Chico: none, Sacramento(90), Redding(73)
Sacramento: Chico(90), San_Francisco(87), none
